src/libs/ReportUtils.js

```diff
@@ -279,11 +279,12 @@ function isSettled(reportID) {
         return false;
     }
     const report = allReports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
-    if ((typeof report === 'object' && Object.keys(report).length === 0) || report.isWaitingOnBankAccount) {
+    if (typeof report === 'object' && Object.keys(report).length === 0) {
         return false;
     }
 
-    return report.statusNum === CONST.REPORT.STATUS.REIMBURSED;
+    // APPROVED status is for Personal policies
```

Comment:
Personal policies are only used for iou reports. For expense report we always use group policies(free, control or collect policy) and we add the status as APPROVED.

So we can update the comment to say 

// APPROVED status is for free group policy

Reply:
Would update this.

Reply:
Done, fixed.